modules/editorial-comments/editorial-comments.php

```diff
@@ -189,261 +190,283 @@ function the_comment_form( ) {
 				<br class="clear" style="margin-bottom:35px;" />
 				<span style="display: none;" class="error"></span>
 			</p>
 
 			<input type="hidden" value="" id="ef-comment_parent" name="ef-comment_parent" />
 			<input type="hidden" name="ef-post_id" id="ef-post_id" value="<?php echo esc_attr( $post->ID ); ?>" />
 
-			<?php wp_nonce_field('comment', 'ef_comment_nonce', false); ?>
+			<?php wp_nonce_field( 'comment', 'ef_comment_nonce', false ); ?>
 
 			<br class="clear" />
 		</div>
 
-		<?php
-	}
-
-	/**
-	 * Maybe display who was notified underneath an editorial comment.
-	 *
-	 * @param int $comment_id
-	 * @return void
-	 */
-	function maybe_output_comment_meta( $comment_id ) {
-		if ( ! $this->module_enabled( 'notifications' ) || ! apply_filters( 'ef_editorial_comments_show_notified_users', true ) ) {
-			return;
-		}
-
-		$notification = get_comment_meta( $comment_id, 'notification_list', true );
-
-		if ( empty( $notification ) ) {
-			$message = esc_html__( 'No users or groups were notified.', 'edit-flow' );
-		} else {
-			$message = '<strong>'. esc_html__( 'Notified', 'edit-flow' ) . ':</strong> ' . esc_html( $notification );
+			<?php
 		}
 
-		echo '<p class="ef-notification-meta">' . $message . '</p>';
-	}
-
-	/**
-	 * Displays a single comment
-	 */
-	function the_comment($comment, $args, $depth) {
-		global $current_user, $userdata;
-
-		// Get current user
-		wp_get_current_user() ;
-
-		$GLOBALS['comment'] = $comment;
+		/**
+		 * Maybe display who was notified underneath an editorial comment.
+		 *
+		 * @param int $comment_id
+		 * @return void
+		 */
+		public function maybe_output_comment_meta( $comment_id ) {
+			if ( ! $this->module_enabled( 'notifications' ) || ! apply_filters( 'ef_editorial_comments_show_notified_users', true ) ) {
+				return;
+			}
 
-		$actions = array();
+			$notification = get_comment_meta( $comment_id, 'notification_list', true );
 
-		$actions_string = '';
-		// Comments can only be added by users that can edit the post
-		if ( current_user_can('edit_post', $comment->comment_post_ID) ) {
-			$actions['reply'] = '<a onclick="editorialCommentReply.open(\''.$comment->comment_ID.'\',\''.$comment->comment_post_ID.'\');return false;" class="vim-r hide-if-no-js" title="'.__( 'Reply to this comment', 'edit-flow' ).'" href="#">' . __( 'Reply', 'edit-flow' ) . '</a>';
+			if ( empty( $notification ) ) {
+				$message = esc_html__( 'No users or groups were notified.', 'edit-flow' );
+			} else {
+				$message = '<strong>' . esc_html__( 'Notified', 'edit-flow' ) . ':</strong> ' . esc_html( $notification );
+			}
 
-			$sep = ' ';
-			$i = 0;
-			foreach ( $actions as $action => $link ) {
-				++$i;
-				// Reply and quickedit need a hide-if-no-js span
-				if ( 'reply' == $action || 'quickedit' == $action )
-					$action .= ' hide-if-no-js';
+			// It's already been escaped above.
+			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
```

Comment:
FYI `phpcs` has an inline way to add comments for a warning:

```suggestion
			// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped -- It's already been escaped above
```

In the [docs under **Ignoring Parts of a File**](https://github.com/squizlabs/PHP_CodeSniffer/wiki/Advanced-Usage#ignoring-parts-of-a-file):

> If required, you can add a note explaining why sniffs are being disable and re-enabled by using the -- separator.
>
> ```php
> // phpcs:disable PEAR,Squiz.Arrays -- this isn't our code
> $foo = [1,2,3];
> bar($foo,true);
> // phpcs:enable PEAR.Functions.FunctionCallSignature -- check function calls again
> bar($foo,false);
> // phpcs:enable -- this is out code again, so turn everything back on
> ```

I typically like putting an explanation on all `phpcs:ignore`/`phpcs:disable`s, but that's probably out of scope for all the changes you're making here.

Reply:
Done